device.go

```diff
@@ -33,34 +42,36 @@ type DeviceResponse struct {
-	IsManagedByServiceInstance odata.Object  `json:"is_managed_by__service_instance,omitempty"`
-	IsManagedByDevice          interface{}   `json:"is_managed_by__device,omitempty"`
-	ShouldBeRunningRelease     *odata.Object `json:"should_be_running__release,omitempty"`
-	Note                       interface{}   `json:"note,omitempty"`
-	LocalID                    interface{}   `json:"local_id,omitempty"`
-	ManagedBySupervisorRelease interface{}   `json:"should_be_managed_by__supervisor_release,omitempty"`
-	ProvisioningProgress       interface{}   `json:"provisioning_progress,omitempty"`
-	DownloadProgress           interface{}   `json:"download_progress,omitempty"`
-	LogsChannel                interface{}   `json:"logs_channel,omitempty"`
-	IsLockedUntil              interface{}   `json:"is_locked_until__date,omitempty"`
-	IsAccessibleBySupportUntil interface{}   `json:"is_accessible_by_support_until__date,omitempty"`
+	CPUId                              string        `json:"cpu_id,omitempty"`
+	SotrageBlockDevice                 string        `json:"storage_block_device,omitempty"`
+	PublicAddress                      string        `json:"public_address,omitempty"`
+	MacAddress                         string        `json:"mac_address,omitempty"`
+	APIHeartbeatState                  string        `json:"api_heartbeat_state,omitempty"`
+	OSVersion                          string        `json:"os_version,omitempty"`
+	OSVariant                          string        `json:"os_variant,omitempty"`
+	SupervisorVersion                  string        `json:"supervisor_version,omitempty"`
+	ProvisioningState                  string        `json:"provisioning_state,omitempty"`
+	Longitude                          string        `json:"longitude,omitempty"`
+	Latitude                           string        `json:"latitude,omitempty"`
+	Location                           string        `json:"location,omitempty"`
+	CustomLongitude                    string        `json:"custom_longitude,omitempty"`
+	CustomLatitude                     string        `json:"custom_latitude,omitempty"`
+	CreatedAt                          string        `json:"created_at,omitempty"`
+	DeviceType                         odata.Object  `json:"device_type,omitempty"`
+	BelongsToApplication               odata.Object  `json:"belongs_to__application,omitempty"`
+	BelongsToUser                      *odata.Object `json:"belongs_to__user,omitempty"`
+	IsManagedByServiceInstance         odata.Object  `json:"is_managed_by__service_instance,omitempty"`
+	IsManagedByDevice                  interface{}   `json:"is_managed_by__device,omitempty"`
+	IsRunningRelease                   *odata.Object `json:"is_running__release,omitempty"`
+	ShouldBeRunningRelease             *odata.Object `json:"should_be_running__release,omitempty"`
+	ShouldBeManagedBySupervisorRelease odata.Object  `json:"should_be_managed_by__supervisor_release,omitempty"`
```

Comment:
There's a mix of `*odata.Object` and `odata.Object`. Should they all be pointers?

Reply:
Yea I agree!